proxy/Plugin.cc

```diff
@@ -299,7 +299,11 @@ plugin_init(bool validateOnly)
         argv[i] = vars[i];
       }
     }
-    argv[argc] = nullptr;
+
+    if (argc < MAX_PLUGIN_ARGS)
```

Comment:
I think we should be consistent, and encourage the use of {} in these if-else. Since we went through the pain of running the clang-tidy with that fix.